dash/frontend/src/app/modules/private/pages/cluster/kube-hunter/kube-hunter.component.html

```diff
@@ -29,9 +29,9 @@ <h3 [ngClass]="penetrationTestStatusInvalid ? 'text-style-red' : 'text-style-gre
                 <th>High</th>
               </tr>
               <tr>
-                <td class="last-run-data">{{ scansExist ? mostRecentVulnerabilities.low : ''}}</td>
-                <td class="last-run-data">{{ scansExist ? mostRecentVulnerabilities.medium : ''}}</td>
-                <td class="last-run-data">{{ scansExist ? mostRecentVulnerabilities.high : ''}}</td>
+                <td class="last-run-data">{{ scansExist ? mostRecentVulnerabilities.low : 'N/A'}}</td>
```

Comment:
Will this show N/A while the page is loading?

Reply:
Yes, are we trying to avoid showing anything while loading in M9sweeper?

Reply:
No, but it looks like a bug if it flashes incorrect information before loading the correct info.

Reply:
It can also be confusing if it loads slowly and says kube-hunter isn't configured when it really is

Reply:
It no longer shows anything here until the page has received an API response in the latest commit.